client/app/states/dashboard/dashboard.state.js

```diff
@@ -129,24 +173,31 @@
 
       if (definedServiceIdsServices.subcount > 0) {
         vm.servicesCount.total = definedServiceIdsServices.subcount;
-        vm.servicesCount.current = definedServiceIdsServices.subcount === 0 ? nonRetiredServices.count :
-        retiredServices.subcount + nonRetiredServices.subcount;
 
-        vm.servicesCount.retired = vm.servicesCount.total - vm.servicesCount.current;
+        vm.servicesCount.retired = retiredServices.subcount;
 
         vm.servicesCount.soon = expiringServices.subcount;
+        
+        vm.servicesCount.current = vm.servicesCount.total - vm.servicesCount.retired - vm.servicesCount.soon;
       }
 
       vm.servicesFeature = true;
     }
 
     vm.requestsFeature = false;
-    if (angular.isDefined(pendingRequests)) {
+
+    if (angular.isDefined(pendingRequests) &&
```

Comment:
Shouldn't this be `||` instead of `&&`? I assume we still want the counts even if only one of them is defined...